connect.sh

```diff
@@ -44,3 +44,7 @@ sudo snap connect ${SNAP_NAME}:process-control     :process-control
 sudo snap connect ${SNAP_NAME}:shutdown            :shutdown
 sudo snap connect ${SNAP_NAME}:dbus-wpa            wpa-supplicant:service
 sudo snap connect ${SNAP_NAME}:home                :home
+
+# Disable old relay-term
```

Comment:
This is the wrong place to take this action.  See the [snap install hook](https://snapcraft.io/docs/supported-snap-hooks).

Reply:
There's a problem in doing this inside `install` hook, how can i get the `snap-name` field?

For example, I want to disable `relay-term` service, I need to do something like this:
`snapctl stop <snap-name>.relay-term --disable`

doing this without the `snap-name` results in following error:
`error: cannot perform the following tasks:
Run install hook of "pelion-edge" snap if present (run hook "install": error: error running snapctl: unknown service: "relay-term")`

Reply:
One solution is to use this: `SNAP_NAME=${SNAP_NAME:-pelion-edge}` variable just the way it is used in `connect.sh` script. Is it a good solution?

Reply:
From within the snap environment, the current snap name can be retrieved from the environment variable [SNAP_INSTANCE_NAME](https://snapcraft.io/docs/environment-variables). 

See here for an example: https://github.com/PelionIoT/snap-pelion-edge/blob/1b90307f37654a8cd54322f2eb3230021917e82a/files/edge-core/scripts/edge-core-factory-reset#L27

The line you add to the install hook will look something like this:
```
snapctl stop --disable ${SNAP_INSTANCE_NAME}.relay-term
```